samples/existingWindowsBackend/mywebapi-windows/charts/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for mywebapi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: azdspublic/mywebapiwindows
```

Comment:
@philon-msft I'm looking into how to achieve this. It might not be possible using VS and the built in publish functionality. It seems to be stripping dashes from my project name.

Reply:
Don't worry about it too much then